torch/_functorch/partitioners.py

```diff
@@ -1273,6 +1291,8 @@ def get_default_op_list() -> OpTypes:
         aten.full,
         aten.as_strided,
         aten.zeros,
+        aten.empty,
```

Comment:
I think plausibly `aten.empty` should be treated even more specially than this. Since `aten.empty` is always free to recompute, regardless of whether or not it's fusible into a downstream op.